native/acter/src/api/account.rs

```diff
@@ -68,7 +65,7 @@ impl Account {
                 let buf = account
                     .get_avatar(MediaFormat::File)
                     .await?
-                    .context("No avatar URL given")?;
+                    .unwrap_or_default();
```

Comment:
what is `default` here? An empty buffer? Then we do not want to return that. We should not be returning empty values of things that are clearly `None`.

Reply:
Fixed